File: ProductService/ProductService.API/DTO/GetProductDTO.cs
namespace ProductService.BLL.DTO;

public class GetProductDto
{
    public required string Title { get; set; }
    public string? Description { get; set; }
    public required decimal Price { get; set; }
    public List<GetProductImageDto> Images { get; set; } = new();
    public required ProviderDto Provider { get; set; }
}

File: FrontEnd/mikeberries-frontend/src/entities/Product/GetProductDto.ts
import type {GetProductImageDto} from "./GetProductImageDto.ts";
import type {ProviderDto} from "../Provider/ProviderDto.ts";

export interface GetProductDto{
    id: string;
    title: string;
    description?: string;
    price: number;
    images: GetProductImageDto[];
    provider: ProviderDto
}
